rllib/core/models/catalog.py

```diff
@@ -314,6 +314,8 @@ def get_encoder_config(
         activation = model_config_dict["fcnet_activation"]
         output_activation = model_config_dict["fcnet_activation"]
         fcnet_hiddens = model_config_dict["fcnet_hiddens"]
+        # TODO (sven): We should only use fcnet_hiddens[-1] for MLP encoders.
```

Comment:
why should we not use fc_hiddens[-1] for lstm/cnn based encoders? Can we update the comment here to address this

Reply:
Fixed the comment. We should move to a new ModuleConfig asap and stop using the old Model(V2)Config dict.